IFC4x3/Properties/t/TransverseInclination_3Cd$wHaE95zeG4Dy9Ad2WJ/Documentation.md

```diff
@@ -0,0 +1 @@
+The angle of inclination of the transverse.
```

Comment:
Geometric property or requirement? It would be good if the documentation specified that.

Reply:
It is a required property. I also change the definition.
Commit : cfaba917732a40e0dda213d95ee3ec4f293d5203